makefiles/optlib2c_input.mak

```
@@ -3,7 +3,8 @@ OPTLIB2C_INPUT = \
 	optlib/RSpec.ctags			\
 	optlib/cmake.ctags			\
 	optlib/ctags-optlib.ctags		\
-	optlib/elm.ctags			\
+	optlib/elixir.ctags		\
+	optlib/elm.ctags		\
```

You should not touch elm.ctags.

---

Yes, this was a conflict when rebasing 09861b6 to `master`.
The tabs in the line `optlib/elm.ctags` were corrected in 806f296, but that commit happened _after_ 09861b6, which triggered the conflict.

I should have seen and corrected the tabs, tho.

---

(trying out Github's suggestions feature, sorry if it's messy)
```suggestion
	optlib/elixir.ctags		\
	optlib/elm.ctags		\
```